Simplenote/src/main/java/com/automattic/simplenote/utils/ReviewAccountVerifier.java

```diff
@@ -0,0 +1,74 @@
+package com.automattic.simplenote.utils;
+
+import android.os.Handler;
+import android.os.Looper;
+
+import com.automattic.simplenote.Simplenote;
+import com.automattic.simplenote.models.Account;
+import com.simperium.client.Bucket;
+import com.simperium.client.BucketObjectMissingException;
+
+import static com.automattic.simplenote.models.Account.KEY_EMAIL_VERIFICATION;
+
+/**
+ * When a user login into their account, Simplenote starts indexing the different buckets including Bucket<Account>.
+ * We setup a OnNetworkChangeListener on the account bucket to listen when the bucket is indexed. One of the parameters
+ * of the callback onNetworkChange called is Bucket.ChangeType. If this parameter is INDEX, then we know that the
+ * account bucket was indexed (synchronized). When the account bucket is synchronized, we can check whether the account
+ * was verified or not
+ */
```

Comment:
In light of the implementation of the class, do you think all of this belongs here as class documentation? Could we explain the behavior and intention of the class differently? It reads like we're explaining the implementation of the class but not its purpose.

```java
/**
 * Monitors account verification status by watching the `account` bucket and notifies of status and changes.
 * 
 * When connecting this monitor will wait until we have received positive confirmation from the
 * server that we're sync'd before reporting the verification status because the account might
 * have (and probably was) updated from another device.
 */
```

Reply:
Agree. Fixed in 79d4813bf85e462852237f08a6c4800f50a713d5.